README.md

```diff
@@ -1,45 +1,124 @@
-
-To determine whether a fix or feature is present in a given release, you will need the commit hash where the fix or feature was added to the `master` branch of the core `react-native` repo. If you know the PR, you can look for the comment from **@facebook-github-bot** that says 'closed this in <COMMIT_HASH>'.
-
-Once you have the commit hash, navigate to `https://github.com/facebook/react-native/commit/<COMMIT_HASH>`. Look closely at the commit message, underneath which you will find a list of tags associated with the commit. These tags will tell you which releases contains this commit. For example, commit [5e80d95e034259af8c41b50756a623756cc81a77](https://github.com/facebook/react-native/commit/5e80d95e034259af8c41b50756a623756cc81a77) has the following tags as of this writing: `v0.55.0-rc.0 v0.54.2 v0.54.1 v0.54.0 v0.54.0-rc.4 v0.54.0-rc.3 v0.54.0-rc.2 v0.54.0-rc.0 latest`. These tags tell us that the commit first made it into the 0.54 release candidate, eventually landing in the 0.54 stable release. It is also present, as you'd expect, in the 0.55 release candidate (and should make it to 0.55 stable, and so on).
-
-If the commit is only present in `master` (i.e. has no tags), then the commit has yet to be picked up by a release (or it may have been included in a follow up cherry pick for a patch version). You can expect it to be included in the next release candidate that is cut once the designed features have all landed.
-
-## Backporting/cherry-picking of changes to existing builds
-
-From time to time, [backporting](https://en.wikipedia.org/wiki/Backporting) a change to an existing release (including candidates) may be needed. Examples include security issues or critical regressions. If you believe a pull request on `react-native` is a candidate for backporting, please mention it in the version associated _backport request_ issue.
-
-Please note that, if the change hasn't landed on `react-native`'s master, it can't be cherry picked to a release tag yet.
+# React Native’s Release Process
+
+At Facebook React Native is not consumed via released versions, instead they
+always consume the latest version of the `master` branch. This means that they
+are able to iterate fast for _their_ needs; if something is broken the fix is a
+single commit away.
+
+External users are of course similarly able to consume the latest `master`
+version, however this is most likely a too fast moving target for most–aside
+from turn around to get a fix in being longer.
+
+To reconcile these two different use-cases, React Native’s release process is a
+different one from the one many users are familiar with in the JavaScript
+ecosystem.
+
+## The process at a high-level
+
+1. A new version branch is branched off of the `master` branch–hereafter
+   referred to as “the stable branch”.
```

Comment:
Yup, much better 👍

Reply:
ah btw - may be worth mentioning here that up to this point the branch has been cut "as close as possible" time wise to when .0 of the previous version in released - as I think it will make it easier to contextualise the proposal for the "new plan"?

Reply:
I think I will mention this in the detailed overview as the last step where it states the process repeats.